### PR TITLE
sane_backends: added PNG and fixed flags

### PR DESCRIPTION
**Pull request by kenmays**
sane-1.0.27: Added PNG. Tested on hrev53452. Passed all tests.
Ref:  #4143

### CODE REVIEW

File: media-gfx/sane_backends/sane_backends-1.0.27.recipe [hunk 1]
```diff
@@ -90,13 +92,13 @@ BUILD()
 		--libdir=$libDir \
 		--includedir=$includeDir \
 		--datadir=$dataDir \
-		--with-docdir=$docDir \
+		--docdir=$docDir \
```
Comment (Begasus) [inline media-gfx/sane_backends/sane_backends-1.0.27.recipe]: Any reason why these are needed? Running "runConfigure" should take care of all the above?

Comment (kenmays) [reply media-gfx/sane_backends/sane_backends-1.0.27.recipe]: Feel free to edit as needed. Eveything is tested as working.

**Comment by extrowerk** [comment]
Builds / packages fine oon 64 bit. No functionality check was done as i got no device.

**Comment by Vidrep** [comment]
Builds on 64 bit. Scans tested up to 1200 dpi. Save as PNM, TIFF, PNG, JPEG working OK.

File: media-gfx/sane_backends/sane_backends-1.0.27.recipe [hunk 2]
```diff
@@ -16,13 +16,11 @@ documentation."
 HOMEPAGE="http://www.sane-project.org"
 COPYRIGHT="David Mosberger-Tang, Andy Beck"
 LICENSE="GNU LGPL v2"
-REVISION="4"
-SOURCE_URI="https://alioth.debian.org/frs/download.php/file/4224/sane-backends-$portVersion.tar.gz"
+REVISION="5"
+SOURCE_URI="https://gitlab.com/sane-project/backends/uploads/6ca779fc5ae2e825efbac91ad7bc75fb/sane-backends-1.0.27.tar.gz"
```
Comment (korli) [inline media-gfx/sane_backends/sane_backends-1.0.27.recipe]: 1.0.27 => $portVersion

File: media-gfx/sane_backends/sane_backends-1.0.27.recipe [hunk 3]
```diff
@@ -59,6 +58,7 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
```
Comment (korli) [inline media-gfx/sane_backends/sane_backends-1.0.27.recipe]: shouldn' be needed.

File: media-gfx/sane_backends/sane_backends-1.0.27.recipe (hunk 2 above)
Comment (kenmays) [reply media-gfx/sane_backends/sane_backends-1.0.27.recipe]: Done

File: media-gfx/sane_backends/sane_backends-1.0.27.recipe (hunk 3 above)
Comment (kenmays) [reply media-gfx/sane_backends/sane_backends-1.0.27.recipe]: Actually, PNG/JPG/TIFF is used in most scanning software.

File: media-gfx/sane_backends/sane_backends-1.0.27.recipe [hunk 4]
```diff
@@ -84,19 +84,12 @@ BUILD()
...
-		--with-docdir=$docDir \
 		--enable-pthread \
-		--disable-latex \
-		--disable-translations \
-		--enable-libusb_1_0 \
 		--enable-rpath \
 		--enable-pnm-backend \
+		--with-usb \
+		--disable-ipv6 \
+		--disable-locking \
```
Comment (korli) [inline media-gfx/sane_backends/sane_backends-1.0.27.recipe]: could you patch acinclude.m4#L333 instead?

Comment (kenmays) [reply media-gfx/sane_backends/sane_backends-1.0.27.recipe]: Possibly, but this is the suggested method (and well tested).

**Pull request closed without merging**